app/services/placeholder_users/delete_service.rb

```diff
@@ -30,6 +30,7 @@
 
 class PlaceholderUsers::DeleteService < ::BaseServices::Delete
   def destroy(placeholder)
+    placeholder.update_column(:status, Principal.statuses[:locked])
```

Comment:
Using `placeholder.locked!` would be easier to understand.

Reply:
Ah, these are autogenerated from the enum then? I only saw the ones in `user.rb`.

Reply:
Yes, they are: https://api.rubyonrails.org/v6.1.2.1/classes/ActiveRecord/Enum.html